modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss

```diff
@@ -59,7 +59,6 @@
 	.autosave-feedback-container {
 		align-items: center;
 		display: flex;
-		justify-content: center;
```

Comment:
@rodrigopaulino can you please check if the behavior described in [LPS-120480](https://issues.liferay.com/browse/LPS-120480) is happening after your changes?

Reply:
Hi, @carolmariaabb.

This is what happens:
<img width="1344" alt="Screen Shot 2022-01-04 at 10 26 34" src="https://user-images.githubusercontent.com/11450348/148065923-a2b168e3-347c-4c2b-a247-73edec4f4499.png">

Not the same behavior, less critical than what's been reported in the past imo, but not ideal either.
I'd say to drop this change from the PR (the message will move around when changing tabs, but that's the current behavior) and open a technical debt to deal with the responsiveness of the navigation bar altogether. Right now, we have too many buttons and no responsive solution for this.

Here's Forms in current 7.4 in an iPhone X simulation:
<img width="160" alt="Screen Shot 2022-01-04 at 10 40 44" src="https://user-images.githubusercontent.com/11450348/148067795-4fe152b1-da47-4a7a-88d5-eb90ae09c1c2.png">

iPad simulation:
<img width="197" alt="Screen Shot 2022-01-04 at 10 41 01" src="https://user-images.githubusercontent.com/11450348/148067818-1787f42f-23ca-4435-bb4b-d42059cd5a81.png">

What do you think?

Reply:
>  open a technical debt to deal with the responsiveness of the navigation bar altogether.

I totally agree with you!

Reply:
Great. It's done. I'll open a technical debt for that. Thanks, @carolmariaabb